modules/vstudio/vs2010_vcxproj.lua

```diff
@@ -2666,6 +2667,13 @@
 	end
 
 
+	function m.fastUpToDateCheck(prj)
+		if prj.fastuptodate == "Disable" then
+			m.element("DisableFastUpToDateCheck", nil, "true")
```

Comment:
Is this available from VS2010, or was it added in VS2019? If the latter, the above check should include `_ACTION >= "vs2019"`.

Reply:
Available since VS2010 I believe.  I've found some only MSDN forum posts about it pretty far back.